adbms/src/test/scala/de/up/hpi/informationsystems/adbms/definition/RecordTest.scala

```diff
@@ -60,13 +60,13 @@ class RecordTest extends WordSpec with Matchers {
       }
 
       "project to itself, when projected by an empty list" in {
-        emptyRecord.project(Seq.empty) should equal(Success(emptyRecord))
+        emptyRecord.project(Set.empty: Set[UntypedColumnDef]) should equal(Success(emptyRecord))
```

Comment:
The change from Seq to Set results in compile-time errors w.r.t. the overloaded `project` method. Apparently `Set` and `Relation` for parameters is ambiguous, so right now I have provide a type when using either version of the method.

This is obviously not desirable and I am looking into alternatives.